### ISSUE TITLE
Printout from IDE not in color as displayed on screen

### ISSUE THREAD

**Issue by waxoil256** [arduino/Arduino#3013]
This would be a useful enhancement

**Comment by Sembazuru** [comment]
I agree, this would be quite useful. I like to print source codes to pdf for viewing elsewhere. Other coding editors that I've used do this and I value this feature.
I have a couple more Printout improvements in mind that I'll put in a new issue to not dilute this one. See issue https://github.com/arduino/Arduino/issues/4145 for my other improvements.

**Comment by benpaddlejones** [comment]
Can this issue be elevated? It's a big problem for teaching as code examples don't look the same in B&W making it hard for students to learn basics.
